lib/graph3d/Graph3d.js

```diff
@@ -1450,6 +1458,15 @@ Graph3d.prototype._hsv2rgb = function(H, S, V) {
   return 'RGB(' + parseInt(R*255) + ',' + parseInt(G*255) + ',' + parseInt(B*255) + ')';
 };
 
+// https://stackoverflow.com/questions/5623838/rgb-to-hex-and-hex-to-rgb
```

Comment:
Could you please use the function `hexToRGB` from the [vis-util package](https://github.com/visjs/vis-util/blob/master/src/util.ts#L911). Thanks!